### PR TITLE
Fix export error messages encoding issues #279

### PR DESCRIPTION
**Pull request by sbihel**
A fix for issue #279. The bug is fully fixed compared to PR #283.

### CODE REVIEW

File: ftplugin/orgmode/plugins/Export.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def _export(cls, format_):
 		p.wait()
 
 		if p.returncode != 0 or settings.get(u'org_export_verbose') == 1:
-			echom('\n'.join(p.communicate()))
+			echom('\n'.join(map(lambda x: x.decode(), p.communicate())))
```
Comment (akstrfn) [inline ftplugin/orgmode/plugins/Export.py]: for consistent decoding use ``u_decode`` function from here [file](https://github.com/jceb/vim-orgmode/blob/master/ftplugin/orgmode/py3compat/encode_compatibility.py)

**Comment by sbihel** [comment]
`u_decode` does nothing for python3

**Comment by akstrfn** [comment]
I know

**Comment by sbihel** [comment]
I am using python3 so it does not fix the bug 🤔 maybe it is more complex than I thought

**Comment by akstrfn** [comment]
I just checked this from my computer and not mobile and you are right ``u_decode`` is useless here. You solution should work.

**Comment by sbihel** [comment]
Do I have to do anything more?

**Comment by jceb** [comment]
Thanks a lot

**Pull request merged**